sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/persons/PersonDirectorySteps.java

```diff
@@ -124,22 +244,90 @@ public PersonDirectorySteps(
         });
 
     When(
-        "I apply on the APPLY FILTERS button",
-        () -> {
-          webDriverHelpers.waitUntilIdentifiedElementIsVisibleAndClickable(
-              APPLY_FILTERS_BUTTON, 30);
-          webDriverHelpers.clickOnWebElementBySelector(APPLY_FILTERS_BUTTON);
-          TimeUnit.SECONDS.sleep(10);
-        });
-
-    When(
-        "I click on the RESET FILTERS button",
+        "I click on the RESET FILTERS button for Person",
         () -> {
           webDriverHelpers.waitUntilIdentifiedElementIsVisibleAndClickable(
               RESET_FILTERS_BUTTON, 30);
-          TimeUnit.SECONDS.sleep(10);
           webDriverHelpers.clickOnWebElementBySelector(RESET_FILTERS_BUTTON);
           TimeUnit.SECONDS.sleep(10);
         });
+
+    Then(
+        "I search after last created person from API by {string}",
+        (String searchCriteria) -> {
+          String searchText = "";
+          String personUUID = apiState.getLastCreatedPerson().getUuid();
+          switch (searchCriteria) {
+            case "uuid":
+              searchText = personUUID;
+              break;
+            case "full name":
+              searchText =
+                  apiState.getLastCreatedPerson().getLastName()
+                      + " "
+                      + apiState.getLastCreatedPerson().getFirstName();
+              break;
```

Comment:
can we cover here all cases so the method will be reusable by everyone on further tests? :D